REFERENCE.md

```diff
@@ -851,472 +857,487 @@ The following parameters are available in the `postgresql::server` class:
 * [`manage_pg_ident_conf`](#manage_pg_ident_conf)
 * [`manage_recovery_conf`](#manage_recovery_conf)
 * [`manage_postgresql_conf_perms`](#manage_postgresql_conf_perms)
+* [`manage_selinux`](#manage_selinux)
 * [`module_workdir`](#module_workdir)
 * [`manage_datadir`](#manage_datadir)
 * [`manage_logdir`](#manage_logdir)
 * [`manage_xlogdir`](#manage_xlogdir)
+* [`password_encryption`](#password_encryption)
 * [`roles`](#roles)
 * [`config_entries`](#config_entries)
 * [`pg_hba_rules`](#pg_hba_rules)
-* [`version`](#version)
-* [`extra_systemd_config`](#extra_systemd_config)
-* [`manage_selinux`](#manage_selinux)
-* [`password_encryption`](#password_encryption)
+
+##### <a name="backup_enable"></a>`backup_enable`
+
+Data type: `Boolean`
+
+Whether a backup job should be enabled.
+
+Default value: `$postgresql::params::backup_enable`
+
+##### <a name="backup_options"></a>`backup_options`
+
+Data type: `Hash`
+
+A hash of options that should be passed through to the backup provider.
+
+Default value: `{}`
+
+##### <a name="backup_provider"></a>`backup_provider`
+
+Data type: `Enum['pg_dump']`
+
+Specifies the backup provider to use.
+
+Default value: `$postgresql::params::backup_provider`
+
+##### <a name="version"></a>`version`
+
+Data type: `Any`
+
+Deprecated. Use postgresql::globals instead. Sets PostgreSQL version
+
+Default value: ``undef``
+
+##### <a name="extra_systemd_config"></a>`extra_systemd_config`
+
+Data type: `Any`
+
+Adds extra config to systemd config file, can for instance be used to add extra openfiles. This can be a multi line string
+
+Default value: `$postgresql::params::extra_systemd_config`
 
 ##### <a name="postgres_password"></a>`postgres_password`
 
 Data type: `Optional[Variant[String[1], Sensitive[String[1]], Integer]]`
 
-Sets the password for the postgres user to your specified value. By default, this setting uses the superuser account in the Postgres database, with a user called postgres and no password.
+
```

Comment:
Looks like you badly broke documentation. That was the reason it was closed previously: https://github.com/puppetlabs/puppetlabs-postgresql/pull/1358#discussion_r951316331

Reply:
Yep, this has been an oversight. v8.2.1 is coming soon. :)